rio_tiler_pds/modis/aws/__init__.py

```diff
@@ -1,4 +1,6 @@
 """rio-tiler-pds.modis.aws"""
 
-from .modis_astraea import MODISReader as MODISASTRAEAReader  # noqa
-from .modis_pds import MODISReader as MODISPDSReader  # noqa
+# fmt: off
+from rio_tiler_pds.modis.aws.modis_astraea import MODISReader as MODISASTRAEAReader # noqa isort:skip
```

Comment:
black, isort and flake8 weren't happy so I had add to:
- tell isort to skip
- tell black to not reformat

Reply:
Weird. Some setting is contradictory among the tools?

Reply:
I don't know why and couldn't find a better way to handle this.